client/src/AllRoutes.js

```diff
@@ -79,18 +82,27 @@ class AllRoutes extends Component {
   }
 
   renderProtectedRoutes = () => {
-    if (this.state.userData === null || this.state.userData === undefined) {
-      this.setState({ redirectUrl: RouteMap.Login, errorMessage: "An error occured: Please try again or contact your administrator for more information." });
-      return "";
-    } else {
+
+    // if (this.state.canLogin === false || this.state.userData === undefined) {
+    //   console.log("checking protected route")
+      
+    //   return (
+    //     <Route path="*" element={<Navigate to={RouteMap.Login} />} key="Navigate" />
+    //   );
+      // this.setState({ redirectUrl: RouteMap.Login, errorMessage: "An error occured: Please try again or contact your administrator for more information." });
+      // return (
+      //   <Route path={RouteMap.Login} exact element={<Login userData={this.state.userData} errorMessage="An error occured: Please try again or contact your administrator for more information." />} key="Login" />
+      // );
+    // } else {
```

Comment:
Remove code that's commented out, thx!

Reply:
modified file to remove comments. please take a look and lmk.